### DISCUSSION TITLE
Architektura dla Datawarehouse

### DISCUSSION THREAD

**Discussion by maciejgos** [AzureCommunityPL/group#194] [Pomoc] [answered]
Cześć,

Dostałem zadanie stworzenia architektury dla DWH 😉.
Wybór padł na Azure Synapse + Data Factory coś na wzór https://github.com/mspnp/azure-data-factory-sqldw-elt-pipeline.

Teraz pytanie jakie będę miał benefity przy wykorzystaniu Data Lake vs Blob Storage bo nie do końca czuję ten temat?
Opis w dokumentacji DL jest trochę dla mnie lakoniczny https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction#key-features-of-data-lake-storage-gen2.

Dodam jeszcze, że źródłem będą dwie bazy MS SQL stojące w on-prem.

Dzięki za podpowiedź

**Comment by maciejgos** [comment, marked as answer]
To już sobie samemu odpowiedziałem na powyższe pytanie. 
Może komuś się przyda na przyszłość:
Data Lake Gen 2 dodaje nam bajery w postaci:
- Wsparcie dla Hadoop
- Hierarchical namespaces
- ACL's, RBAC, AAD
- zostało zaprojektowane pod big data
- ma dużą skalowalność oraz wydajność
- reszta funkcjonalności Blob Storage

**Comment by piotrgwiazda** [comment]
ADLS powinien się lubić ze Sparkiem, który siedzi w Synapsie (nie jest
tani, ale zintegrowany). Gdybyś chciał ładować dane Sparkiem pomiędzy
bronze/silver/gold (np. provisioned SQL pool jako gold a parkiety jako
niższe warstwy). W przypadku użycia external table z ALDS Gen2 działa
TYPE=Hadoop.

śr., 13 sty 2021, 18:45 użytkownik Maciej Gos <notifications@github.com>
napisał:
